pkg/deploy/taskdir/definitions/def_0_3_test.go

```diff
@@ -985,6 +1030,24 @@ func TestDefinitionToUpdateTaskRequest_0_3(t *testing.T) {
 						Type:  "shorttext",
 						Regex: "foo.*",
 					},
+					{
+						Name: "Config var",
+						Slug: "config_var",
+						Type: "configvar",
+						Default: map[string]interface{}{
+							"config": "API_KEY",
+						},
```

Comment:
Why a map with a `config` field instead of a singular string value?

```suggestion
						Default: "API_KEY",
```

My take:
1. We should use the `type` field as a hint that informs how to parse the default value.
2. Once we support object parameter types, how do we identify this is a config not a map with a `"config"` field?
3. There's a case for consistency since we already use a dedicated `config` field in (a) env vars and (b) option values. But, IMO we should merge `config+value` in (b) as it can use `type` as in (1). OTOH, (a) has a good reason as it can't use (1) and does need to distinguish.

Reply:
Was there a specific reason we went with the `config` field instead of using `value`? Didn't see a discussion of that on the original PR: https://github.com/airplanedev/lib/pull/94

If not, let's go with strings instead of maps here. Let's merge the `config` field on select options in with `value` sometime soon.